vendor/vendor.json

```diff
@@ -1,6 +1,6 @@
 {
 	"comment": "",
-	"ignore": "test",
+	"ignore": "test appengine",
```

Comment:
Because we have un-vendored deps relative to appengine. I think that it's a sub deps but I haven't found/search for it. I don't even know if we (plan to) support appengine. So this could be a good place to discuss about it ^^